snap7/server.py

```diff
@@ -287,3 +289,25 @@ def clear_events(self):
         """
         logger.debug("clearing event queue")
         return self.library.Srv_ClearEvents(self.pointer)
+
+
+def mainloop(tcpport: int = 1102):
```

Comment:
May be a stupid question, but why is the mainloop running with port 1102? IsnÄt 102 the S7comm used port?

Reply:
because then you dont need to run the test suite as root.